src/programs/match_template/match_template.cpp

```diff
@@ -52,7 +64,9 @@ class
     ArrayOfAggregatedTemplateResults aggregated_results;
     bool                             is_rotated_by_90 = false;
 
-    float GetMaxJobWaitTimeInSeconds( ) { return 360.0f; }
+    float GetMaxJobWaitTimeInSeconds( ) { return 120.0f; }
```

Comment:
I would prefer leaving this at 360. Should not be necessary, but I do observed this timeout every now and then, especially when running on over 100 GPUs.

Reply:
Will revert.